libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/AppGlobals.java

```diff
@@ -10,6 +10,15 @@
 
 public class AppGlobals {
 
+    public static final String MAPBOX_ACCESS_CODE = "pk.eyJ1IjoibW96aWxsYS13ZWJwcm9kIiwiYSI6ImNqMml4MXNhaTAwMjgzM28yYXRqbjlmZmkifQ.OfL6NY7noQth9mbbN_tDbA";
```

Comment:
There's nothing to worry about. I've created that access code specifically for all MozStumbler usage, independent of the app store / way of downloading / building it.